### ISSUE TITLE
Demo would be nice

### ISSUE THREAD

**Issue by shtefcs** [robconery/meteor-shop#6]
Hi there,

Demo would be awesome if you had on repo documentation. I think that every repo should have it by default, cause it's give users faster and better insight what is in there.

Peace,

**Comment by dsinghr** [comment]
http://rocket-shop.meteor.com/

**Comment by shtefcs** [comment]
Thank you @dsinghr .

How experienced you are with javascript and meteor ?

**Comment by robconery** [comment]
Would love a PR if you had time for the README :)

**Comment by odyright** [comment]
Hi there,
1) Demo doesn't work!
2) Is this repo suitable for production use?

**Comment by robconery** [comment]
Yeah it’s a bit out of date, no it’s not really ready for production use. You can probably tweak a few things to get it going, however.

**Comment by odyright** [comment]
hello,
thank you @robconery anyway it looks great.

Secondly I would like to know according to the opinion of an expert: for a site's e-commerce site that wants to be strong and effective, which library to choose? I want to say between Laravel(PHP) and meteor(NODE JS)?
